mne/preprocessing/ica.py

```diff
@@ -544,16 +544,14 @@ def _fit(self, data, max_pca_components, fit_type):
         """Aux function."""
         random_state = check_random_state(self.random_state)
 
+        from sklearn.decomposition import PCA
         if not check_version('sklearn', '0.18'):
-            from sklearn.decomposition import RandomizedPCA
-            # XXX fix copy==True later. Bug in sklearn, see PR #2273
-            pca = RandomizedPCA(n_components=max_pca_components, whiten=True,
-                                copy=True, random_state=random_state)
-
+            # fix copy==True later. Bug in sklearn, see PR #2273
```

Comment:
also this comment has to do with `RandomizedPCA`:

https://github.com/scikit-learn/scikit-learn/pull/2273

So it can be removed